src/SprykerEco/Yves/FactFinderWebComponents/Theme/default/components/atoms/campaign/campaign.twig

```diff
@@ -0,0 +1,25 @@
+{% extends model('component') %}
+
+{% define config = {
+    name: 'campaign',
+    tag: 'campaign'
+} %}
+
+{% define data = {
+    properties: {
+        component: '',
```

Comment:
Why do we need these properties? As I see only `feedbacktext` is used.

Reply:
I removed all not use data from all molecules and atoms